.gitignore

```diff
@@ -8,4 +8,5 @@ id_rsa
 id_rsa.pub
 mondoo-test-secret-key
 gha-creds-*.json
-/tools
\ No newline at end of file
+/tools
+.goreleaser-edge.yml
```

Comment:
We must ignore the copied file with the altered config such that goreleaser doesn't complain for creating a release from a dirty checkout